apps/src/templates/teacherDashboard/shapes.jsx

```diff
@@ -42,4 +42,4 @@ export const loadErrorShape = PropTypes.shape({
   message: PropTypes.string.isRequired,
 });
 
-export const OAuthSectionTypes = makeEnum('google_classroom', 'clever');
+export const OAuthSectionTypes = makeEnum('google_classroom', 'clever', 'microsoft_classroom');
```

Comment:
Microsoft Classroom work will be scheduled for later. My spec was showing the eventual goal.